configs/envoy_service_to_service.template.json

```diff
@@ -300,6 +341,17 @@
       "refresh_delay_ms": 30000
     },
 
+    "cds": {
+      "cluster": {
+        "name": "cds",
+        "connect_timeout_ms": 250,
+        "type": "strict_dns",
+        "lb_type": "round_robin",
+        "features": "http2",
```

Comment:
I would probably del `features: http2` from CDS and RDS that you added for simplicity.